### ISSUE TITLE
Events not firing in Chrome/Chromium since v42

### ISSUE THREAD

**Issue by acs** [fat/bean#125]
HI!

We are using flotr2 to draw the charts in our dashboards. And flotr2 uses bean for events management.

Since Chrome/Chromium v42 the events are not fired when we are over a chart (mouse up, mouse down ...).

This is how flotr2 uses bean:

https://github.com/HumbleSoftware/Flotr2/blob/master/js/EventAdapter.js

I have tested it with current bean master.

**Comment by ded** [comment]
interesting. and you can confirm that there was no problems with V41

**Comment by acs** [comment]
Yes, I have playing with different version using the debian packages at:

http://mirror.pcbeta.com/google/chrome/deb/pool/main/g/google-chrome-stable/

for isolating the problem. In v41 flotr2 events are working with and old bean version and also with master.

With v42 flotr2 events are NOT working with the old bean version and also not working with current master.

In you need I test anything in order to make easier for you to isolate the problem, I will help! :)

**Comment by acs** [comment]
Any news about this issue?

Thanks!
